docs/LogGuide.md

```diff
@@ -1972,21 +2176,30 @@ Parsed Log Line Examples:
 
 <!-- AUTO-GENERATED-CONTENT:END -->
 
-It seems likely that this line was added in order to extend functionality
-for the [NetworkBuff](#line26),
-[NetworkBuffRemove](#line30),
-and [NetworkActionSync](#line37)
-log lines without breaking previous content or plugins.
+This line conveys all current status effects on an entity.
+This can be useful if a plugin or overlay was started after zoning in.
+
+#### Data Fields
+
+This is a variable-length line.
+It can expand up to 30 status effects.
+Beginning with the field called `data3`, each status effect takes three fields.
+
+The first data field for each Trio is some unknown data plus the effect ID in the latter 2 bytes.
```

Comment:
The next significant byte is the same as `count`, e.g. this text applies to it as well:
> The rightmost byte of the flag is the "value", usually treated as a stack count,
> but may be employed for other purposes by specific status effects.

Example `|0106FD|426AD0F1|400034DC|`, `01` = stacks of effect `0x06FD`, `Vulnerability Up`.
And two stacks: `|0206FD|426D22D8|400034D8|`.

The field is defined as a `ushort` in the packet, but I can't recall seeing any data in the leftmost byte off the top of my head.

Reply:
Done